kubectl-plugin/pkg/cmd/create/create_workergroup.go

```diff
@@ -0,0 +1,165 @@
+package create
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/spf13/cobra"
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+
+	"github.com/ray-project/kuberay/kubectl-plugin/pkg/util/client"
+
+	rayv1 "github.com/ray-project/kuberay/ray-operator/apis/ray/v1"
+
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	cmdutil "k8s.io/kubectl/pkg/cmd/util"
+	"k8s.io/kubectl/pkg/util/templates"
+)
+
+type CreateWorkerGroupOptions struct {
+	configFlags       *genericclioptions.ConfigFlags
+	ioStreams         *genericclioptions.IOStreams
+	clusterName       string
+	groupName         string
+	rayVersion        string
+	image             string
+	workerCPU         string
+	workerMemory      string
+	workerReplicas    int32
+	workerMinReplicas int32
+	workerMaxReplicas int32
+}
+
+var (
+	createWorkerGroupLong = templates.LongDesc(`
+		Adds a worker group to an existing RayCluster.
+	`)
+
+	createWorkerGroupExample = templates.Examples(`
+		# Create a worker group in an existing RayCluster
+		kubectl ray create worker-group gpu-group --cluster sample-cluster --image rayproject/ray:2.39.0 --worker-gpu 1 --worker-memory=5Gi
```

Comment:
GPU in example here, but I don't see it as part of the options.

Reply:
Assuming we're waiting on #2675

Reply:
Removed --worker-group for now, we can re-add after #2675